datanode/gateway/graphql/oracles_resolver.go

```diff
@@ -76,9 +77,9 @@ func resolveTrigger(obj any) (trigger TriggerKind) {
 		switch trig := obj.(type) {
 		case *vegapb.EthCallTrigger_TimeTrigger:
 			if trig.TimeTrigger != nil {
-				init := int64(trig.TimeTrigger.GetInitial())
+				init := strconv.FormatUint(trig.TimeTrigger.GetInitial(), 10)
```

Comment:
RFC3339 encoded should be:

```suggestion
				init := time.Unix(trig.TimeTrigger.GetInitial(), 0).Format(time.RFC3339)
```